tests/boardfarm_plugins/run_bf.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+scriptdir=$(dirname "$(readlink -f "${0}")")
+bf_plugins_dir=${scriptdir}
+bf_dir=$(realpath "${scriptdir}/../../../boardfarm/boardfarm")
```

Comment:
I would put this script one directory higher, in `tests/run_bf.sh`. It will be the typical entrypoint for the developer who wants to run tests, so each subdirectory is more typing.